repository/CargoPackageManager-Tests.package/CGOSolverTest.class/instance/noDependenciesPackageBigger.st

```diff
@@ -0,0 +1,15 @@
+helpers
+noDependenciesPackageBigger
+	
+	^ CGOPackageVersion newFrom: 
+		( Dictionary new
+			add: #package -> #'NoDependencies';
```

Comment:
This one should have package name  #'NoDependenciesBigger' else it will conflict with the package defined in #noDependenciesPackage